deploy/olm-catalog/3scale-operator/3scale-operator.v0.4.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,296 @@
+    * **Capabilities** Ability to define 3scale API definitions and set them into a 3scale API Management solution
+
+    ### Documentation
+    [3scale api management](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management)
+    [Deploying 3scale using the operator](https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.7/html/installing_3scale/onpremises-installation#deploying-threescale-using-the-operator)
+
+    ### Getting help
+    If you encounter any issues while using 3scale operator, you can create an issue on our [Github repo](https://github.com/3scale/3scale-operator) for bugs, enhancements, or other requests.
+
+    ### Contributing
+    You can contribute by:
+
+    * Raising any issues you find using 3scale Operator
+    * Fixing issues by opening [Pull Requests](https://github.com/3scale/3scale-operator/pulls)
+    * Improving [documentation](https://github.com/3scale/3scale-operator/blob/v0.4.0/doc/user-guide.md)
+    * Talking about 3scale Operator
+
+    All bugs, tasks or enhancements are tracked as [GitHub issues](https://github.com/3scale/3scale-operator/issues).
+
+    ### License
+    3scale Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/3scale-operator/blob/master/LICENSE)
+
+  keywords:
+    - 3scale
+    - API
+  displayName: 3scale
+  install:
+    spec:
+      deployments:
+      - name: 3scale-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: threescale-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: threescale-operator
+            spec:
+              containers:
+              - command:
+                - 3scale-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: threescale-operator
+                - name: BACKEND_IMAGE
+                  value: registry.stage.redhat.io/3scale-amp2/backend-rhel7:3scale2.7
+                - name: APICAST_IMAGE
+                  value: registry.stage.redhat.io/3scale-amp2/apicast-gateway-rhel7:3scale2.7
+                - name: SYSTEM_IMAGE
+                  value: registry.stage.redhat.io/3scale-amp2/system-rhel7:3scale2.7
+                - name: ZYNC_IMAGE
+                  value: registry.stage.redhat.io/3scale-amp2/zync-rhel7:3scale2.7
+                - name: SYSTEM_MEMCACHED_IMAGE
+                  value: registry.stage.redhat.io/3scale-amp2/memcached-rhel7:3scale2.7
+                image: registry.stage.redhat.io/3scale-amp2/3scale-rhel7-operator:1.10-5
+                imagePullPolicy: Always
```

Comment:
Because the tag is not floating we can remove the imagePullPolicy:Always field and set the default one as we did with  the community operator